bufferserver/src/main/java/com/datatorrent/bufferserver/policy/RandomOne.java

```diff
@@ -53,7 +54,8 @@ private RandomOne()
   @Override
   public boolean distribute(Set<PhysicalNode> nodes, SerializedData data) throws InterruptedException
   {
-    int count = (int)(Math.random() * nodes.size());
+    Random rand = new Random();
```

Comment:
-1: there is no performance improvement as `Math.random()` delegates to `Random` `nextDouble()` method anyway. The change leads to unnecessary `Random()` instance creation.

Reply:
The intent of the policy is to distribute tuples randomly each time. Using a single instance should work fine and Math.random() is doing that internally anyway.